google-cloud-spanner/lib/google/cloud/spanner/service.rb

```diff
@@ -545,12 +547,13 @@ def list_backup_operations instance_id,
 
         def restore_database backup_instance_id, backup_id,
                              database_instance_id, database_id,
-                             call_options: nil
+                             call_options: nil, encryption_config: nil
```

Comment:
The restore database `encryption_config` is different than the `backup` and the `create database` encryption configs. See https://github.com/googleapis/googleapis/blob/master/google/spanner/admin/database/v1/spanner_database_admin.proto#L740-L741 and https://github.com/googleapis/googleapis/blob/master/google/spanner/admin/database/v1/spanner_database_admin.proto#L744-L778. The encryption here can be one of the following:

- [CustomerManagedEncryption](https://github.com/googleapis/googleapis/blob/master/google/spanner/admin/database/v1/spanner_database_admin.proto#L761): a key should be specified.
- [GoogleDefaultEncryption](https://github.com/googleapis/googleapis/blob/master/google/spanner/admin/database/v1/spanner_database_admin.proto#L757): no key should be specified.
- [UseBackupEncryption](https://github.com/googleapis/googleapis/blob/master/google/spanner/admin/database/v1/spanner_database_admin.proto#L754): no key should be specified.